man/ctags.1.rst.in

```diff
@@ -1321,17 +1321,30 @@ OPTION ITEMS
 	Limits the depth of directory recursion enabled with the ``--recurse``
 	(``-R``) option.
 
-``--options=file|directory``
-	Read additional options from file or directory. If a file is specified,
-	it should contain one option per line. If a directory is specified
-	(and scandir function is available at build configuration time), files
-	suffixed with .ctags or .conf under the directory are read. (On MSDOS or
-	MSWindows this directory traverse feature is temporary disable because
-	the contributor of this feature has no access to the platforms.
-	Volunteers are welcome). As a special case, if "--options=NONE" is
-	specified as the first option on the command line, it will disable
-	the automatic reading of any configuration options from either a file
-	or the environment (see "FILES").
+``--optlib-dir=[+]directory``
+	Adds a directory to or reset **optlib** path list.
+	By default, optlib path list is empty.
+
+``--options=pathname``
+	Read additional options from file or directory.
+
+	@CTAGS_NAME_EXECUTABLE@ searches *pathname* in optlib path list
+	first. If @CTAGS_NAME_EXECUTABLE@ cannot find a file or directory
+	in the list, @CTAGS_NAME_EXECUTABLE@ reads a file or directory
+	at the specified *pathname*.
+
+	If a file is specified, it should contain one option per line. If
+	a directory is specified, files suffixed with ".ctags" under it
+	are read in alphabetical order.
+
+	As a special case, if "--options=NONE" is specified as the first
+	option on the command line, preloading is disabled; the option
+	will disable the automatic reading of any configuration options
+	from either a file or the environment (see "FILES").
+
+``--options-maybe=pathname``
+	Does the same as ``--options`` but doesn't make an error if file
```

Comment:
Could replace "Does the same" with "Same" and "make an error" with "cause an error".